tests/01-ingress-http-traffic/ing-smoketest.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+kubectl create namespace ingress-smoketest
+
+sleep 3
+
+kubectl -n ingress-smoketest apply -f ./ingress-smoketest.yaml
+
+sleep 3
+
+output=$(curl -s -o /dev/null -w "%{http_code}" https://iingress-smoketest-app.apps.cloud-platform-test-1.k8s.integration.dsd.io)
```

Comment:
Is that a spelling mistake (`iingress`) - should it be

`https://ingress-smoketest-app.apps.cloud-platform-test-1.k8s.integration.dsd.io`

Reply:
Yes, that was the test scenario that was not changed back - fixed